src/shared/components/mutationTable/column/GeneColumnFormatter.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import {IColumnFormatterProps, IColumnFormatterData, IColumnFormatter}
+    from "../../enhancedReactTable/IColumnFormatterProps";
+
+/**
+ * @author Selcuk Onur Sumer
+ */
+export default class GeneColumnFormatter extends React.Component<IColumnFormatterProps, {}> implements IColumnFormatter
+{
+    public static sortFunction(a:IColumnFormatterData, b:IColumnFormatterData):boolean
+    {
+        let aValue = GeneColumnFormatter.getTextValue(a);
+        let bValue = GeneColumnFormatter.getTextValue(b);
+
+        return aValue > bValue;
+    }
+
+    public static filterValue(data:IColumnFormatterData):string
+    {
+        return GeneColumnFormatter.getTextValue(data);
+    }
+
+    /**
+     * Default text value for a gene is its hugo gene symbol.
+     *
+     * @param data  column formatter data
+     * @returns {string}    hugo gene symbol
+     */
```

Comment:
@adufilie would it be worth making a type hugoSymbol so this would be self documenting? (where hugoSymbol was just a string)